tests/spec/fs.mknod.spec.js

```diff
@@ -1,82 +1,94 @@
     var fs = util.fs();
     expect(fs.mknod).to.be.a('function');
     done();
   });
 
-  it('should return an error if part of the parent path does not exist', function(done) {
+  it('should return an error if part of the parent path does not exist', function (done) {
     var fs = util.fs();
 
-    fs.mknod('/dir/mydir', 'DIRECTORY', function(error) {
+    fs.mknod('/dir/mydir', 'DIRECTORY', function (error) {
       expect(error.code).to.equal('ENOENT');
       done();
     });
   });
 
-  it('should return an error if path already exists', function(done) {
+  it('should return an error if path already exists', function (done) {
     var fs = util.fs();
 
-    fs.mknod('/', 'DIRECTORY', function(error) {
+    fs.mknod('/', 'DIRECTORY', function (error) {
       expect(error.code).to.equal('EEXIST');
       done();
     });
   });
 
-  it('should return an error if the parent node is not a directory', function(done) {
+  it('should return an error if the parent node is not a directory', function (done) {
     var fs = util.fs();
 
-    fs.mknod('/file', 'FILE' , function(error) {
-      if(error) throw error;
+    fs.mknod('/file', 'FILE', function (error) {
+      if (error) throw error;
 
-      fs.mknod('/file/myfile', 'FILE', function(error) {
+      fs.mknod('/file/myfile', 'FILE', function (error) {
         expect(error.code).to.equal('ENOTDIR');
         done();
       });
     });
   });
 
-  it('should return an error if the mode provided is not DIRECTORY or FILE', function(done) {
+  it('should return an error if the mode provided is not DIRECTORY or FILE', function (done) {
     var fs = util.fs();
 
-    fs.mknod('/symlink', 'SYMLINK', function(error) {
+    fs.mknod('/symlink', 'SYMLINK', function (error) {
       expect(error).to.exist;
       expect(error.code).to.equal('EINVAL');
       done();
     });
   });
 
-  it('should make a new directory', function(done) {
+  it('should make a new directory', function (done) {
     var fs = util.fs();
 
-    fs.mknod('/dir', 'DIRECTORY', function(error) {
-      if(error) throw error;
+    fs.mknod('/dir', 'DIRECTORY', function (error) {
+      if (error) throw error;
 
-      fs.stat('/dir', function(error, stats) {
+      fs.stat('/dir', function (error, stats) {
         expect(error).not.to.exist;
         expect(stats.type).to.equal('DIRECTORY');
         done();
       });
     });
   });
 
-  it('should make a new file', function(done) {
+  it('should make a new file', function (done) {
     var fs = util.fs();
 
-    fs.mknod('/file', 'FILE' , function(error) {
-      if(error) throw error;
-      
-      fs.stat('/file', function(error, result) {
+    fs.mknod('/file', 'FILE', function (error) {
+      if (error) throw error;
+
+      fs.stat('/file', function (error, result) {
         expect(error).not.to.exist;
         expect(result.type).to.equal('FILE');
         done();
       });
     });
   });
+  describe.only('Promise test mknod', function () {
```

Comment:
I think it's OK to keep your newly added test inside the existing `describe` block - other students have been putting them there!